apache2/waf_logging/waf_log_util.cc

```diff
@@ -162,7 +162,7 @@ static std::string get_json_log_message(const char* timestamp, const char* resou
     #define MANDATORY_RULE_MESSAGE_LENGTH (sizeof(MANDATORY_RULE_MESSAGE) - 1)
     std::array<char, 1024> mandatory_message {MANDATORY_RULE_MESSAGE};
     str_view message_str;
-    if (messages) {
+    if (messages && strlen(messages) != 0) {
```

Comment:
minor - we should identify the cases in which the len is 0 and take care of it like with pcre